src/lib.rs

```diff
@@ -114,8 +114,8 @@ pub const VERSION: Version = Version {
     } else {
         ""
     },
-    xdr_curr: env!("XDR_CURR_VERSION"),
-    xdr_next: env!("XDR_NEXT_VERSION"),
+    xdr_curr: include_str!("../xdr/curr-version"),
+    xdr_next: include_str!("../xdr/next-version"),
```

Comment:
Taking a page out of @tsachiherman's approach to implementing this in the soroban-rpc project 👏🏻